### DISCUSSION TITLE
Multi-rack devices

### DISCUSSION THREAD

**Discussion by krasotinpa** [netbox-community/netbox#11743] [General]
Hi all!

Please advise me, how you model devices in NETBOX, such as UPS, that occupy more than one rack? Like this - [Liebert EXL S1](https://www.vertiv.com/de-emea/products-catalog/critical-power/uninterruptible-power-supplies-ups/liebert-exl-s1-ups-300-1250-kw/)
I can split it in 3 parts and place each part in separate rack.
But maybe there is better way?

**Comment by jeremystretch** [comment]
If it's truly a monolithic device, I would just model it as a free-standing device within the site/location, eschewing racks (as it doesn't appear to actually mount within a rack).
